phpstan.neon

```
@@ -0,0 +1,10 @@
+parameters:
+    level: max
+    reportUnmatchedIgnoredErrors: false
```

```suggestion
    reportUnmatchedIgnoredErrors: true
```
And probably then we'll see the ignored error in line 10 should be removed.